apparmor.d/profiles-m-r/mutt

```diff
@@ -0,0 +1,184 @@
+
+  # Might be able to get away without this
+  owner /tmp/mutt* lrwk,
+
+  # Needed for the edit operation.
+  @{HOME}/ r,
+
+  # User mbox
+  owner /var/{spool/,}mail/* rwk,
+  owner @{HOME}/postponed    rwk,
+  owner @{HOME}/sent         rwk,
+  # User maildir
+  owner @{HOME}/{M,m}ail/    rw,
+  owner @{HOME}/{M,m}ail/**  rwk, 
+
+  # Trusted certificate store
+  owner @{HOME}/.mutt_certificates rwk,
+  
+  # Mutt history file
+  owner @{HOME}/.mutthistory rwk,
+  
+  # Mutt signature file
+  owner @{HOME}/.signature r,
+
+  # Common location for mail aliases
+  owner @{HOME}/.mail_aliases r,
+  
+  /usr/share/terminfo/** r,
+
+  /etc/mime.types r,
+ 
+  # Mutt mailcap search path
+  owner @{HOME}/.mailcap    r,
+  /etc/{mutt/,}mailcap      r,
+  /usr/{local/,}etc/mailcap r,
+
+  # Used to exec programs defined in the mailcap.
+  # There are countless programs that can be executed from the mailcap.
+  # This profile includes only the most basic.
+  /{usr/,}bin/{,ba,da}sh  rix,
+ 
+  /{usr/,}{s,}bin/sendmail rPUx, 
+  /{usr/,}bin/ispell       rPUx, 
+  # TODO: Add a profile for abook (Most distros don't ship this anymore though)
+  /{usr/,}bin/abook        rPUx, 
+ 
+  /{usr/,}bin/w3m             rCx -> html-renderer,
+  /{usr/,}bin/lynx            rCx -> html-renderer,
+  /{usr/,}bin/vim             rCx -> editor,
+  /{usr/,}bin/sensible-editor rCx -> editor,
+  /{usr/,}bin/more            rCx -> pager,
+  /{usr/,}bin/less            rCx -> pager,
+  /{usr/,}bin/pager           rCx -> pager,
+  /{usr/,}bin/gpg{2,}         rCx -> gpg, 
+  /{usr/,}bin/pgpewrap        rCx -> gpg,
+  
+
+  profile html-renderer {
+    include <abstractions/base>
+
+    /{usr/,}bin/w3m     mrix,
+    /{usr/,}bin/lynx    mrix,
+    
+    owner @{HOME}/.w3m/* rw,
+
+    owner /tmp/mutt* rw,
+
+    include if exists <local/mutt_html-renderer>
+  }
+
+  profile editor {
+    include <abstractions/base>
+    include <abstractions/nameservice-strict>
+
+    /{usr/,}bin/sensible-editor        mr,
+    /{usr/,}bin/vim                  mrix,
+    /{usr/,}bin/vim.*                mrix,
+    /{usr/,}bin/{,ba,da}sh            rix,
+    /{usr/,}bin/which{,.debianutils}  rix,
+
+    /usr/share/vim/{,**} r,
+    /usr/share/terminfo/** r,
+
+    /etc/vimrc r,
+    /etc/vim/{,**} r,
+
+    owner @{HOME}/.selected_editor r,
+    owner @{HOME}/.viminfo{,.tmp} rw,
+    owner @{HOME}/.vimrc r,
+  
+    # Vim swap file 
+    owner @{HOME}/ r,
+    owner @{HOME}/.cache/ r,
+    owner @{HOME}/.cache/vim/** wr,
+   
+    # This is the file that holds the message
+    owner /tmp/mutt* rw,
+
+    include if exists <local/mutt_editor>
+  }
+  
+  profile pager {
+    include <abstractions/base>
+    include <abstractions/consoles>
+  
+    /{usr/,}bin/less  mr,
+    /{usr/,}bin/more  mr,
+    /{usr/,}bin/pager mr,
+    
+    owner @{HOME}/.lesshsQ r,
+    owner @{HOME}/.local/state/ r,
+    owner @{HOME}/.local/state/less* rw,
+
+    /usr/share/terminfo/x/xterm-256color r,
```

Comment:
Use `/usr/share/terminfo/**` instead (color definition change depending of the terminal)